src/lib/operators/table_scan.cpp

```diff
@@ -0,0 +1,220 @@
+#include "table_scan.hpp"
+
+#include <memory>
+
+#include "resolve_type.hpp"
+#include "storage/table.hpp"
+
+namespace opossum {
+
+TableScan::TableScan(const std::shared_ptr<const AbstractOperator>& input_operator, const ColumnID column_id,
+                     const ScanType scan_type, const AllTypeVariant search_value)
+    : _input_operator{input_operator}, _scan_type{scan_type}, _column_id{column_id}, _search_value{search_value} {}
+
+ColumnID TableScan::column_id() const { return _column_id; }
+
+ScanType TableScan::scan_type() const { return _scan_type; }
+
+const AllTypeVariant& TableScan::search_value() const { return _search_value; }
```

Comment:
You could also test those getters to increase your test coverage

Reply:
We decided not to test getters due to their simplicity.